sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOperation.cs

```diff
@@ -307,7 +308,7 @@ public override Pageable<AnalyzeHealthcareEntitiesResultCollection> GetValues(Ca
             //    {
             //        Response<HealthcareJobState> jobState = _serviceClient.HealthStatusNextPage(nextLink, cancellationToken);
 
-            //        AnalyzeHealthcareEntitiesResultCollection result = Transforms.ConvertToAnalyzeHealthcareEntitiesResultCollection(jobState.Value.Results, _idToIndexMap);
+                    AnalyzeHealthcareEntitiesResultCollection result = ExtractHealthcareActionResult(jobState.Value.Results, _idToIndexMap);
```

Comment:
I uncommented the lines where the new function is called. I thought it will be easier for visualization purposes